Merge 19.3.0.1 beta and editorialized release notes into `trunk`

mokagio opened the pull request:
Includes:

- Add GIF support to notifications and comments ([#17981](https://github.com/wordpress-mobile/WordPress-iOS/pull/17981)) by @emilylaguna 

Update: This now also includes the release notes update.

Commit:
Add the animated gif text attachment view provider

Commit:
Add a custom image wrapper for gif images

Commit:
Don't resize gifs

Commit:
Use the AnimatedGifAttachmentViewProvider if it's registered as a handler

Commit:
Register UITextView gif handling using AnimatedGifAttachmentViewProvider

Commit:
Allow gifs in the web comments to play inline

Commit:
Fix lint issues

Commit:
Fix content extension build issue

Commit:
Update AnimatedImageWrapper comment

Commit:
Update RELEASE-NOTES with animated gif support

Commit:
Merge branch 'release/19.3' into try/add-gif-playing-support

Commit:
Move GIF release notes from 19.4 to 19.3

Commit:
Merge pull request #17981 from wordpress-mobile/try/add-gif-playing-support

Add animated gif support to notifications, comments, and my site comments

mokagio committed:
Update translations

mokagio committed:
Update metadata translations

mokagio committed:
Bump version number

mokagio committed:
Update Jetpack metadata translations

mokagio committed:
Restore 19.4 header in `RELEASE-NOTES.txt`

It got lost in 45094b3c101a7f15cdc2634968a0f1ce2bec37c4.

mokagio committed:
Update `release_notes.txt` files with #17981 addition

mokagio reviewed (commented):
- [x] Updates to the localization files (`.strings` and Fastlane metadata) automatically pulled for the new translations that have already been approved in GlotPress
- [x] Version update in `.xcconfig`
- [x] Diffs from the PRs that made it into this beta

mokagio committed:
Add editorialized release notes for WordPress 19.3

mokagio committed:
Add editorialized release notes for Jetpack 19.3

mokagio committed:
Update metadata strings

mokagio committed:
Update metadata strings

mokagio commented on WordPress/Jetpack/Resources/release_notes.txt:
```diff
@@ -1,5 +1,11 @@
-* [*] Add "Copy Link" functionality to Posts List and Pages List [#17911]
-* [***] Enables the ability to use and create WordPress.com sites, and enables the Reader tab. [#17914, #17948]
-* [*] Block editor: Additional error messages for media upload failures. [#17971]
+Three, two, one, liftoff—the Jetpack app now allows you to use and create WordPress.com sites and use the same Reader features as the WordPress app. The sky’s the limit.
```
I got confused by the "use" verb.

#17914 says: "👁️ Notice on the site switcher all of your connected sites are displayed, not just Jetpack ones"

Maybe a clearer wording would be:

> use all of your WordPress.com sites and create new ones, too

But, then we'd have _another_ and conjunction "use all ... and create ... and use the same ..." I think that would make the sentence worse off.

So, all in all, I think this is fine to ship with.

AliSoftware replied:
What about this?
```suggestion
Three, two, one, liftoff—the Jetpack app now allows you to use all of your WordPress.com sites, create new ones, and use the same Reader features as the WordPress app. The sky’s the limit.
```

mokagio reviewed (commented):
- [x] `AppStoreStrings.po(t)` changed with editorialized release notes

AliSoftware commented on WordPress/Jetpack/Resources/release_notes.txt:
```diff
@@ -1,5 +1,11 @@
+
+When you add a new Insights card on the Stats screen, your screen view will jump down to show your brand-new card.
+
+You can now copy links to published, scheduled, and draft posts and pages using the Copy Link button above the Trash button. We also translated the button text into Turkish. Delightful.
```
> We also translated the button text into Turkish. Delightful.

Now I'm hungry for Turkish delights 😋 

Btw, I'm not sure why we specifically mention Turkish translation here while it will be translated in many languages, not just `tr`.
I think it's because the contributor initially provided the Turkish translation by directly adding the string to the `tr.lproj/Localizable.strings` in their PR? But that ends up being overridden by our GlotPress translation pipeline anyway, and as you can see by looking at the various `*.lproj/Localizable.strings` in that PR, yes `tr.lproj` ended up being part of the translations already approved in GlotPress for "Copy Link", but they are far from the only language for which we got a translation for that new string.

I'd thus suggest to remove that last sentence from this paragraph of the notes.

AliSoftware commented on WordPress/Resources/release_notes.txt:
```diff
@@ -1,4 +1,9 @@
+
+When you add a new Insights card on the Stats screen, your screen view will jump down to show your brand-new card.
+
+You can now copy links to published, scheduled, and draft posts and pages using the Copy Link button above the Trash button. We also translated the button text into Turkish. Delightful.
```
Same remark as the one in the Jerpack release notes for this

AliSoftware reviewed (commented):
2 suggestions for changing the wording that would otherwise seem quite misleading.

Apart from that wording, the rest of the content of the PR diff seems as expected, so I'll approve to not block the merge, but disable auto-merge to give you a chance to address the wording suggestions.

PS: I plan on looking at the Jetpack Installable Builds job failures on CI today (same certificates expiration issue as last couple of days after the renewal)

AliSoftware commented:
> PS: I plan on looking at the Jetpack Installable Builds job failures on CI today (same certificates expiration issue as last couple of days after the renewal)

Finally fixed Jetpack Installable Builds, see paaHJt-32G-p2#comment-5836 for details.

mokagio committed:
Make release notes regarding Jetpack support for WordPress.com clearer

mokagio committed:
Remove Turkish translation mention from release notes.

@AliSoftware raised a good point here:
https://github.com/wordpress-mobile/WordPress-iOS/pull/18011#discussion_r813817864

> Yes `tr.lproj` ended up being part of the translations already
> approved in GlotPress for "Copy Link", but they are far from the only
> language for which we got a translation for that new string.  # Please
> enter the commit message for your changes.

This would set a precedent for an explosion of translations updates in
the release notes. I think we should save those only for when we add
support for a new locale.

mokagio committed:
Update metadata strings

mokagio committed:
Update metadata strings
